unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,90 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test: device D400*
+
+import time
+import pyrealsense2 as rs
+from rspy import test
+
+# This test checks Timestamp domain toggle option for color and depth sensor
+
+
+def frame_callback(frame):
+    global expected_time_stamp_domain
+    test.check_equal(int(frame.get_frame_timestamp_domain()), expected_time_stamp_domain)
+
+
+################################################################################################
+test.start("Start test")
+
+ENABLE = 1
+DISABLE = 0
+
+devices = test.find_first_device_or_exit()
+
+time.sleep(3)  # The device starts at D0 (Operational) state, allow time for it to get into idle state
+depth_sensor = devices.first_depth_sensor()
+color_sensor = devices.first_color_sensor()
+
+depth_sensor.get_option(rs.option.global_time_enabled)
+color_sensor.get_option(rs.option.global_time_enabled)
+
+depth_profile = next(p for p in depth_sensor.profiles if p.fps() == 30
+                     and p.stream_type() == rs.stream.depth
+                     and p.format() == rs.format.z16
+                     and p.as_video_stream_profile().width() == 640
+                     and p.as_video_stream_profile().height() == 480)
+
+color_profile = next(p for p in color_sensor.profiles if p.fps() == 30
+                     and p.stream_type() == rs.stream.color
+                     and p.format() == rs.format.yuyv
+                     and p.as_video_stream_profile().width() == 640
+                     and p.as_video_stream_profile().height() == 480)
+
+depth_sensor.open(depth_profile)
+depth_sensor.start(frame_callback)
+
```

Comment:
Here it's better to sleep ~ 3 sec to assure you are getting frames before waiting for them..
You can do either this or when you set the data, implement some kind of python conditional variable..

something like
```
timer t(3)

While not t.expired():
   If got frame?
           Good, check it
   not?
       sleep(0.2)
```

If you need help let me know..

Reply:
sleep 3 seconds added to test function.